spec/models/organization_spec.rb

```diff
@@ -114,6 +114,18 @@
 
       expect(organization).not_to be_valid
     end
+
+    it 'is valid with email_settings' do
+      organization.email_settings = ['invoice.finalized', 'credit_note.created']
+
+      expect(organization).to be_valid
+    end
+
+    it 'is invalid with non permitted email_settings value' do
+      organization.email_settings = ['email.not_permitted']
+
+      expect(organization).not_to be_valid
```

Comment:
what about checking that the error is on `email_settings` instead of just not valid?